packages/app-desktop/gui/ProfileManagementScreen.tsx

```diff
@@ -0,0 +1,277 @@
+import * as React from 'react';
+import ButtonBar from './ConfigScreen/ButtonBar';
+import { _ } from '@joplin/lib/locale';
+
+const { connect } = require('react-redux');
+const { themeStyle } = require('@joplin/lib/theme');
+import bridge from '../services/bridge';
+import dialogs from './dialogs';
+import { ProfileConfig, Profile } from '@joplin/lib/services/profileConfig/types';
+import { deleteProfileById, saveProfileConfig } from '@joplin/lib/services/profileConfig';
+import Setting from '@joplin/lib/models/Setting';
+import shim from '@joplin/lib/shim';
+import Logger from '@joplin/utils/Logger';
+import { CSSProperties } from 'react';
+import { AppState } from '../app.reducer';
+
+const logger = Logger.create('ProfileManagementScreen');
+
+interface Props {
+	themeId: number;
+	style: CSSProperties;
+	// eslint-disable-next-line @typescript-eslint/ban-types -- Changing types for these variables would be too big of a refactoring
+	dispatch: Function;
```

Comment:
```suggestion
	dispatch: Dispatch;
```
Types: `Dispatch` can be imported `from 'redux';`